src/ui/deleteAccount/ViewDeleteAccountRequest.qml

```diff
@@ -100,10 +103,9 @@ MZInAppAuthenticationBase {
         MZLinkButton {
             Layout.fillWidth: true
 
-            fontName: MZTheme.theme.fontBoldFamily
+            fontName: MZTheme.theme.fontFamily
```

Comment:
this is actually still the wrong font family. the default one set in [MZLinkButton.qml](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/3206e02e69e43eb0c556a8ca0335ca0d558fe5ab/nebula/ui/components/MZLinkButton.qml#L15) is correct, so you can remove this line altogether

Reply:
Thanks, good catch. Updated.